src/JWT.Extensions.AspNetCore/JwtAuthenticationOptions.cs

```diff
@@ -8,6 +8,11 @@ namespace JWT.Extensions.AspNetCore
 {
     public class JwtAuthenticationOptions : AuthenticationSchemeOptions
     {
+        public JwtAuthenticationOptions()

```

Comment:
I'd like initialization into the property rather than adding a ctor for the purpose of a single initialization

Reply:
Because we aren't and can't use auto-properties - we have do initialize in the ctor.